packages/expo-image/ios/EXImageView.m

```diff
@@ -17,9 +36,67 @@ - (void)dealloc
 
 - (void)setSource:(NSDictionary *)source
 {
-  NSURL *imageURL = [RCTConvert NSURL:source[sourceUriKey]];
+  _source = source;
+  _needsReload = YES;
+}
 
-  [self sd_setImageWithURL:imageURL];
+- (void)setResizeMode:(RCTResizeMode)resizeMode
+{
+  if (_resizeMode == resizeMode) return;
```

Comment:
Let's write out full-sized `if` with brackets (💅)

Reply:
You're right, will do 👍